Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -38,6 +38,7 @@ open class AVFoundationPlayback: Playback {
     }
     fileprivate var timeObserver: Any?
     fileprivate var asset: AVURLAsset?
+    private var lastDvrAvailability: Bool?
```

Comment:
@dezouzart , can you help me by creating a test to cover a scenario that use this variable?
Example: `Only triggers didChangeDvrAvailability when lastDvrAvailability is different`